upgrade-notes/backend_20240917_130712.md

```diff
@@ -0,0 +1,3 @@
+#### add complaint detail url to mail templates ([#3445](https://github.com/shopsys/shopsys/pull/3445))
+
+-   see #project-base-diff to update your project
```

Comment:
if the urldecode is necessary, shouldn't we mention it as an instruction for upgrading?